src/resources/assets/sass/main.scss

```diff
@@ -94,3 +94,9 @@
         margin-bottom: 0;
     }
 }
+
+.overlay svg {
+    position: absolute;
+    top: 0;
+    left: 0;
+ }
```

Comment:
Since you already use the scoped style for the dismiss image component, you should put this class there, too (because the name is so generic that it could override some other class.)

Reply:
If I move this class to the scoped style area, the class does not work any more.
Or did I misunderstand you?

Reply:
No that's exactly what I meant. You probably have to add the `::v-deep` there, too.